tests/unit/multi_select_test.coffee

```diff
@@ -0,0 +1,96 @@
+multiSelect = null
+dispatcher = null
+
+module "[Unit] Multi-Select Component unit tests",
+  setup: ->
+    Ember.run ->
+      dispatcher = Ember.EventDispatcher.create()
+      dispatcher.setup()
+      multiSelect = Ember.Widgets.MultiSelectComponent.create
+        content: ['foo', 'bar', 'barca', 'baz']
+
+  teardown: ->
+    Ember.run ->
+      dispatcher.destroy()
+      multiSelect.destroy()
+
+test 'Test preparedContent after some options are already selected', ->
+  expect 2
+  multiSelect.set 'selections', ['bar', 'baz']
+  multiSelect.set 'query', 'ba'
+  equal(multiSelect.get('preparedContent').length, 1)
+  equal(multiSelect.get('preparedContent')[0], 'barca')
+
+test 'Test keyboard and mouse interaction', ->
+  expect 17
+  selectedText = null
+
+  Ember.run ->
```

Comment:
do we really need `Ember.run`? I don't understand enough about testing here, but I was able to remove it in an Iverson test I wrote recently.